aten/src/ATen/gen_backend_select_register.py

```diff
@@ -0,0 +1,48 @@
+from code_template import CodeTemplate
```

Comment:
It would be very nice if this file had a comment on top saying what it does

Reply:
added a comment.